testsuite/manualmode/src/test/java/org/wildfly/core/test/standalone/mgmt/ManagementInterfaceResourcesCommunityTestCase.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright The WildFly Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.wildfly.core.test.standalone.mgmt;
+
+import static org.jboss.as.controller.client.helpers.Operations.createAddress;
+import static org.jboss.as.controller.client.helpers.Operations.createWriteAttributeOperation;
+import static org.jboss.as.test.integration.management.util.ServerReload.executeReloadAndWaitForCompletion;
+
+import org.jboss.as.test.integration.management.util.ServerReload;
+import org.jboss.as.version.Stability;
+import org.jboss.dmr.ModelNode;
+import org.wildfly.common.function.ExceptionRunnable;
+import org.wildfly.core.testrunner.ManagementClient;
+import org.wildfly.core.testrunner.ServerSetupTask;
+import org.wildfly.test.stability.StabilityServerSetupSnapshotRestoreTasks;
+
+/**
+ * Test case to test resource limits and clean up of management interface connections.
+ *
+ * This test case uses attributes defined directly on the HTTP management interface resource for configuration.
+ *
+ * @author <a href="mailto:darran.lofthouse@jboss.com">Darran Lofthouse</a>
+ */
+public class ManagementInterfaceResourcesCommunityTestCase extends AbstractManagementInterfaceResourcesTestCase {
```

Comment:
Should not this test be excluded from the Bootable JAR set? I guess saving a snapshot in a Bootable JAR would not be a valid operation

Reply:
TBH I am not sure, as the process is only running once it would make sense if we could run bootable jar tests otherwise we will never be able to test features with a stability level lower than default.